buildpacks/gradle/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Buildpack API version changed from `0.9` to `0.10`. ([#662](https://github.com/heroku/buildpacks-jvm/pull/662))
```

Comment:
Is it worth mentioning the resultant increase in minimum `lifecycle` version (here and in the other `CHANGELOG.md`s too), like here? :-)
https://github.com/heroku/buildpacks-python/blob/de378063c83e1eec6302e79f93f189162b5bbbbf/CHANGELOG.md?plain=1#L23

Reply:
Added in 1c2715b